### PR TITLE
Eco update 1.1 and minor stuff

### PR DESCRIPTION
**Pull request by nokele**
## Description

Eco and stuff: 
-    NCR, Legion and Shopkeeper get 1 random blueprint,
-    1 random blueprint in Oasis dungeon.
-    Few gun parts in other dungeons.
-    Changed recipes to require less gunpowder (for blueprints).
-    Moved part vending machine to train station, changed price to 300.
-   Guns spawn with 1 magazine instead of 2 now as per vote.

Other:
-    Updates to general’s uniform sprites

Hopefully didn't change anything else this time.
## Motivation and Context
Trying to add more blueprints so the weapons parts are usable when no shopkeep is around.

## How Has This Been Tested?
Compiles and runs locally.

### CODE REVIEW

File: code/game/objects/effects/spawners/f13lootdrop.dm [hunk 1]
```diff
@@ -1372,8 +1372,8 @@
 	/obj/item/book/granter/trait/pa_wear = 1,
 	/obj/item/book/granter/trait/gunsmith_one = 50,
 	/obj/item/book/granter/trait/gunsmith_two = 20,
-	/obj/item/book/granter/trait/gunsmith_three = 10,
-	/obj/item/book/granter/trait/gunsmith_four = 10)
+	/obj/item/book/granter/trait/gunsmith_three = 19,
+	/obj/item/book/granter/trait/gunsmith_four = 1)
```
Comment (Laus4Deus) [inline code/game/objects/effects/spawners/f13lootdrop.dm:1376]: Leave the rates to what they were for the gunbooks

Comment (nokele) [reply code/game/objects/effects/spawners/f13lootdrop.dm:1376]: nice forgot again

**Comment by Laus4Deus** [comment]
don't forget to freshen your fork!

**Comment by nokele** [comment]
New bayonets for the following guns:

- Laser musket
- AER9
- Assault rifle
Better bayonet for the following gun:

- Mosin nagant

**Comment by Laus4Deus** [comment]
I want you to do a separate pr for the bayonets!
We can take it slow, be more methodical about it and get sprites that accommodate to the guns! maybe the shotguns don't really look ok with sprites, maybe the assault carbine needs X = -3 Y = 5 and the assault rifle X = 1 Y = -2.

So, we slow down the process and make sure it is nice

**Comment by nokele** [comment]
Done, probably?

**Pull request merged**